usr/sbin/containerbuild-regionsrv

```diff
@@ -0,0 +1,109 @@
+#!/usr/bin/python3
+# -*- encoding: utf-8 -*-
+
+# Copyright (c) 2020 SUSE LLC.
+#
+# This library is free software; you can redistribute it and/or
+# modify it under the terms of the GNU Lesser General Public
+# License as published by the Free Software Foundation; either
+# version 3.0 of the License, or (at your option) any later version.
+# This library is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+# Lesser General Public License for more details.
+# You should have received a copy of the GNU Lesser General Public
+# License along with this library.
+
+from cloudregister.registerutils import get_instance_data, get_smt, get_config, get_credentials
```

Comment:
Use
`import cloudregister.registerutils as utils`
style and then prefix calls with `utils.` to match the style in other parts of the code.

Reply:
:heavy_check_mark: